virttest/libvirt_xml/libvirt_cpu.py

```diff
@@ -0,0 +1,110 @@
+"""
+Module simplifying manipulation of CPU model and topology part described at
+http://libvirt.org/formatdomain.html
+"""
+
+
+import logging
+
+from virttest.libvirt_xml import vm_xml
+
+
+def set_vm_attrs(vmxml, vm_attrs):
```

Comment:
I still need your help to move this function to virttest/utils_libvirt/libvirt_vmxml.py which is designed for common vm xml setting, not specific feature related.

Reply:
Updated